chapters/syntax.tex

```diff
@@ -55,7 +55,7 @@ \section{Lexical conventions}\doublelabel{lexical-conventions}
   has structured comments in the form of annotations and string
   comments.
 \item
-  Each description-string or string in annotations (= STRING with production annotation-comment in the
+  Each description-string or string in annotations (= STRING with production annotation-clause in the
```

Comment:
It's not clear to me what, "(= STRING with production annotation-clause in the grammar)", actually means.

Reply:
I also noticed that when looking through the text. My current guess is that it should be "within" instead of "with".